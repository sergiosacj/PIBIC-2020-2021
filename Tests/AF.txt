problem    ARp   Ipopt
ARGAUSS    0     0
ARGLCLE    0     0
ARGLINA    2     2
ARGLINB    8     3
ARGTRIG    0     4
BARD       22    8
BDVALUE    0     1
BEALE      17    19
BIGGS6     18    12
BOX3       14    14
BROWNAL    21    5
BROWNBS    90    8
BROWNDEN   11    8
BROYDN3D   0     5
BRYBND     11    17
CHEBYQAD   14    54
FREUROTH   8     13
GULF       11    50
HELIX      25    25
INTEGREQ   0     3
JENSMP     11    10
KOWOSB     15    23
MEYER3     34    47
OSBORNEA   24    37
OSBORNEB   11    24
PENALTY1   10    24
PENALTY2   13    10
POWELLBS   0     12
POWELLSG   21    17
ROSENBR    20    45
SROSENBR   6     13
VARDIM     47    28
WATSON     49    17
WOODS      38    84
