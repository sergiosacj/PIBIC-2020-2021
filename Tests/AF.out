problem    ARp    Ipopt
BOX3       26     14
BEALE      32     19
ARGLINB    8      3
WOODS      16     84
BROWNAL    22     5
BROWNBS    144    8
WATSON     52     17
ARGLINA    4      2
KOWOSB     998    23
INTEGREQ   0      3
BARD       38     8
VARDIM     60     28
SROSENBR   12     13
HELIX      42     25
BDVALUE    0      1
BROYDN3D   0      5
ROSENBR    14     45
POWELLSG   42     17
